app/js/home/home.js

```diff
@@ -13,6 +15,8 @@ let homeModule = angular.module('home', [ uiRouter, 'openmrs-contrib-uicommons']
             template: "<home></home>"
         })
     })
+    .service('ListService', ListService)
```

Comment:
Please do not append "Service" by default as per https://github.com/rkorytkowski/angular-styleguide/blob/master/a1/README.md#factory-and-service-names

I'd call it "ToDo".